<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像处理工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>打开图像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sharpenButton">
        <property name="text">
         <string>锐化图像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="grayButton">
        <property name="text">
         <string>灰度化</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cannyButton">
        <property name="text">
         <string>Canny边缘检测</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gammaButton">
        <property name="text">
         <string>伽马变换</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openSecondImageButton">
        <property name="text">
         <string>选择第二张图像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stitchButton">
        <property name="text">
         <string>图像拼接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="blendButton">
        <property name="text">
         <string>图像融合</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="textureTransferButton">
        <property name="text">
         <string>纹理迁移</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="faceBeautifyButton">
        <property name="text">
         <string>人脸美颜</string>
        </property>
       </widget>
      </item>
      </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightColumnLayout">
      <item>
       <widget class="QPushButton" name="swapImagesButton">
        <property name="text">
         <string>交换主/第二图像</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="imageDisplayLayout">
        <item>
         <widget class="QLabel" name="imageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>请加载第一张图像</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="secondImageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>请加载第二张图像</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
